staging/src/k8s.io/apiserver/pkg/apis/auditregistration/validation/validation.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	audit "k8s.io/apiserver/pkg/apis/audit/validation"
+	auditconf "k8s.io/apiserver/pkg/apis/auditregistration"
+)
+
+// ValidateAuditSink validates the AuditSinks
+func ValidateAuditSink(ac *auditconf.AuditSink) field.ErrorList {
```

Comment:
missing apimachineryvalidation.ValidateObjectMeta call

Reply:
nvmd. We do that from the generic registry. Looks like we spread some calls in the code base that are unnecessary.